flink-formats/flink-json/src/test/java/org/apache/flink/formats/json/JsonRowDataSerDeSchemaTest.java

```diff
@@ -626,6 +635,62 @@ void testSerializationDecimalEncode() throws Exception {
         assertThat(scientificDecimalResult).isEqualTo(scientificDecimalJson);
     }
 
+    @TestTemplate
+    public void testSerDeMultiRowsWithNullValuesIgnored() throws Exception {
```

Comment:
Since we have migrated to Junit5, `public` is no longer needed, see more here: https://docs.google.com/document/d/1514Wa_aNB9bJUen4xm5uiuXOooOJTtXqS_Jqk9KJitU/edit#heading=h.4iilyfqpoiw2
```suggestion
    void testSerDeMultiRowsWithNullValuesIgnored() throws Exception {
```